### ISSUE TITLE
Reverse Axis

### ISSUE THREAD

**Issue by eselmeister** [eclipse/swtchart#1]
For the display of NMR data, the axis needs to be shown in reverse order. Positive x values left, negative x values right.

**Comment by laeubi** [comment]
i'm currently investigating an approach that allows for custom formating of axis. With that feature enabled, one can simply provide a Formatter that inverts the sign in front of the number, and add values simply in reverse order.

**Comment by yoshitaka1008** [comment]
I would like to double-check the requirement.
Does the picture below match with your expectation?

![reversed_axis](https://user-images.githubusercontent.com/11504728/51264212-47bfb880-19b6-11e9-9faf-cb8a7c49a0b4.png)

**Comment by eselmeister** [comment]
It looks good and seems to match the expectations.

**Comment by yoshitaka1008** [comment]
What do you think about adding the following APIs to IAxis?
```java
public interface IAxis {
	...

	/**
	 * Sets the axis reversed. When X axis is reversed, axis values increase from right to left.
	 * When Y axis is reversed, axis values increase from top to bottom.
	 * 
	 * @param reverse
	 *            true to show reversed axis
	 */
	void setReversed(boolean reverse);

	/**
	 * Gets the state indicating if axis is reversed.
	 * 
	 * @return true if axis is reversed
	 */
	boolean isReversed();
}
```

**Comment by eselmeister** [comment]
Yep, that is a nice solution.Am 17.01.2019 17:38 schrieb yoshitaka <notifications@github.com>:What do you think about adding the following APIs to IAxis?
public interface IAxis {
	...

	/**
	 * Sets the axis reversed. When X axis is reversed, axis values increase from right to left.
	 * When Y axis is reversed, axis values increase from top to bottom.
	 * 
	 * @param reverse
	 *            true to show reversed axis
	 */
	void setReversed(boolean reverse);

	/**
	 * Gets the state indicating if axis is reversed.
	 * 
	 * @return true if axis is reversed
	 */
	boolean isReversed();
}

**Comment by yoshitaka1008** [comment]
I have created a pull request to support reversed axis. Please check if the changes make sense.

**Comment by eselmeister** [comment]
Look good. Thanks :-)
